create-budget/azuredeploy.parameters.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "$schema": "https://schema.management.azure.com/schemas/2018-05-01/subscriptionDeploymentTemplate.json#",
```

Comment:
revert this change

Reply:
Why ? I'm intending this to be a budget created at the subscription level.

Reply:
this is not a valid schema for a parameters file

Reply:
Ah! never mind. reverting.